cypress/support/page-objects/ManualEventSyncPageObject.js

```diff
@@ -0,0 +1,48 @@
+import { dataTest } from "../utils";
+import ManualSyncPageObject from "./common/ManualSyncPageObject";
+
+class ManualEventSyncPageObject extends ManualSyncPageObject {
+    constructor(cy) {
+        super(cy, "events");
+    }
+
+    open() {
+        super.open("/#/sync/events");
+        return this;
+    }
+
+    displayOrgUnitChildren(orgUnit) {
+        this.cy
+            .get(dataTest("DialogContent-events-synchronization"))
+            .contains(orgUnit)
+            .parent()
+            .parent()
+            .contains("▸")
+            .click();
+        return this;
+    }
+
+    selectEvent(event) {
+        this.cy
```

Comment:
To realize this action, I think you can use selectRow function of the base class.

Reply:
I have tried it but it does not work, it does not find the texts of the rows of events.